src/form-elements/index.jsx

```diff
@@ -110,7 +110,7 @@ class TextInput extends React.Component {
       <div style={{ ...this.props.style }} className={baseClasses}>
         <ComponentHeader {...this.props} />
         <div className="form-group">
-          <ComponentLabel {...this.props} />
+          <ComponentLabel {...this.props}  className={'form-label'}/>
```

Comment:
I don't think adding className to every ComponentLable is good idea.
Can you make change here https://github.com/Kiho/react-form-builder/blob/master/src/form-elements/component-label.jsx#LL8C5-L8C46 to 
```js
<label className={props.className || 'form-label'}>
```